libeos-updater-util/ostree-bloom.c

```diff
@@ -275,11 +275,11 @@ ostree_bloom_maybe_contains (OstreeBloom   *bloom,
 
   for (i = 0; i < bloom->k; i++)
     {
-      gsize idx;
+      guint64 idx;
 
       idx = bloom->hash_func (element, i);
 
-      if (!ostree_bloom_get_bit (bloom, idx % (bloom->n_bytes * 8)))
+      if (!ostree_bloom_get_bit (bloom, (gsize) (idx % (bloom->n_bytes * 8))))
```

Comment:
I'm guessing that we assume here that `bloom->n_bytes * 8` will never overflow here (perhaps a range check in `ostree_bloom_new` is necessary?)

Reply:
Ah, indeed. I thought we had one of those, but I was wrong. I’ll push a fixup. Thanks for checking thoroughly. :-)

Reply:
Fixup pushed. We don’t need it on `eos3.3` since it doesn’t fix any demonstrable bugs.